app/views/catalog/_show_sections/_default_contents.html.erb

```diff
@@ -25,10 +25,6 @@
           content_ds.ng_xml.xpath('/contentMetadata/resource').each_with_index do |resource, index| %>
             <li class="resource">
               <span class="label">Resource (<%=index+1%>)</span> <%= resource['type']%>
-              <%if can_manage && version_open
-                edit_resource_link="/items/#{params[:id]}/resource/?edit=true&resource=#{resource['id']}"%>
-                <%= link_to "(edit)", edit_resource_link, :data => {ajax_modal: 'trigger'} %>
-              <%end%>
```

Comment:
The `version_open = can_close_version? object.pid` statement can also be deleted, right?